venues/NeurIPS.cc/2021/Conference/python/build-conflicts.py

```diff
@@ -0,0 +1,235 @@
+import openreview
+import argparse
+from datetime import datetime
+
+
+def get_profile_info(profile):
+    
+    domains = set()
+    emails=set()
+    relations = set()
+    publications = set()
+    common_domains = ['gmail.com', 'qq.com', '126.com', '163.com', 
+                      'outloook.com', 'hotmail.com', 'yahoo.com', 'aol.com', 'msn.com', 'ymail.com', 'googlemail.com', 'live.com']
+
+    ## Institution section, get history within the last three years
+    for h in profile.content.get('history', []):
+        if h.get('end') is None or int(h.get('end')) > 2017:
```

Comment:
I noticed that some users don't have an active history in their profile, is it ok to don't take in account any institution?

Reply:
If they don't have a history, we use their email domains. See line 32-34. Does this solve the problem?

Reply:
That could work but sometimes the email domain is a gmail account so there will be profiles with no domains to check. 

I think if there is no domain I would use at least the domain of the history no matter if the year is before 2018. But we can check this with Alina.